src/model/transaction/__tests__/insertFragmentIntoContentState-test.js

```diff
@@ -32,33 +34,42 @@ const DEFAULT_BLOCK_CONFIG = {
   key: 'j',
   type: 'unstyled',
   text: 'xx',
-  characterList: List.of(EMPTY, EMPTY),
   data: new Map({a: 1}),
 };
 
 const initialBlock = contentState.getBlockMap().first();
 
-const createFragment = (fragment = {}) => {
+const createFragment = (fragment = {}, experimentalTreeDataSupport = false) => {
+  const ContentBlockNodeRecord = experimentalTreeDataSupport
+    ? ContentBlockNode
+    : ContentBlock;
   fragment = Array.isArray(fragment) ? fragment : [fragment];
 
   return BlockMapBuilder.createFromArray(
     fragment.map(
       config =>
-        new ContentBlock({
+        new ContentBlockNodeRecord({
           ...DEFAULT_BLOCK_CONFIG,
           ...config,
         }),
     ),
   );
 };
 
+const createContentBlockNodeFragment = fragment => {
```

Comment:
I think instead of using this createContentBlockNodeFragment and the above one we should instead refactor this file to be a bit more DRY and consise on the tests.

The stubs for tree data are not meant to be exactly the same, example its a waste of snapshot to always include the `data` property in all ContentBlockNodes